profile.md

```diff
@@ -12,9 +12,53 @@ openastronomy
 ## Organization name
 OpenAstronomy
 
-## Organization description OpenAstronomy is a collaboration between open source
+## Public Profile
+
+### Website URl
+http://openastronomy.org/
+
+### Tagline
+
+*A very short description*
+
+Confederation of open source projects in Astronomy
+
+### Logo
+http://openastronomy.github.io/img/logo/logoOA_svg.png
+
+### Primary Open Source License
+BSD 3-Clause "New" or "Revised" License (BSD-3-clause)
+(MIT for astrojulia)
+
+### Organization Category
+Science and Medicine
+
+###Technology Tags
+python, julia, c, numpy
+
+### Topic tags
+science, solar physics, astronomy, visualisation, atomic physics
+
+### Ideas List
+http://openastronomy.org/gsoc/gsoc2016/ideas.html
+
+## Organization description 
+
+*These descriptions will be displayed on the
+organization list page (Short Description) and on your organization's page (Long
+Description). More details. The Long Description may include limited Markdown.*
+
+### short (180)
+
+OpenAstronomy is a collaboration between open source
+astronomy and astrophysics projects. Formed at the moment by the
+Astropy, SunPy, yt, ChiantiPy and  astrojulia projects.
+
+### long (2000)
```

Comment:
This description needs to be update to astrojulia (@kbarbary) and to include other activities as workshops that we've been doing (@cadair could you ask @drewleonard42)